020_up_L22.py

```
@@ -0,0 +1,128 @@
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('file', type=str, help='video file')
+    parser.add_argument('title', type=str, nargs='?', help='post title')
+    parser.add_argument('time', type=int, nargs='?', help='time for pre segment', default=0)
+    parser.add_argument('-c, --config', type=str, dest='config', help='change the configuration file path')
+    args = parser.parse_args()
+
+    load_dotenv(args.config)
+    tmpdir = path.dirname(path.abspath(__file__)) + '/tmp'
```

Ruff (0.14.5)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)